### ISSUE TITLE
Active tab too tall

### ISSUE THREAD

**Issue by simoncarne** [Izheil/Quantum-Nox-Firefox-Dark-Full-Theme#149]
**Describe the bug**
Since upgrading to Firefox 96, the active tab is now taller than the rest (and all the tabs are appearing more spaced out than they did with the previous version that I was running - 29 Oct 21, I believe). I have installed the most recent version of MultiRow Tabs (1 Jan 22), but that has made no difference. 

**To Reproduce**
Running Firefox 96, together with MultiRowTab-scrollable.uc. 

**Expected behavior**
I expected the active tab to be the same size as the rest. 

**Screenshots**
In the screenshot below, the active tab is on the right-hand end of the top row (arrowed). 

![Screenshot 2022-01-13 081431](https://user-images.githubusercontent.com/33954467/149291652-ff6a251e-1943-4eee-9c98-e51acf5a8f4e.jpg)

**Desktop** (please complete the following information):

    OS: Windows 10
    Version: Firefox 96 (64 bit)

**Comment by Izheil** [comment]
To me multirow works correctly even after the lastest update, but I see that you are using some other code to change the shape of tabs and their sizing, which is most likely the issue here.

You will have to check which part of the custom code you have is creating the issue, not much that I can do about it without knowing that.

Multirow tabs inherits the tab height from a global variable and uses that to calculate the sizing of tabs to calculate the max number of rows to show, if you want to force a tab height, you can change the value of "inherit" in this part of the code to the value in px that you want it to be:

``` CSS
#TabsToolbar {
        --tab-min-height: inherit !important;
    }
```

For example, for normal density the min value you can use without having issues on tabs is 29px, in the example you'd change it to:

``` CSS
#TabsToolbar {
        --tab-min-height: 29px !important;
    }
```

If you are using compact mode, the smallest size you can give the tabs is 20px of height.

Still, the problem is coming from some issue of other custom code you have to change the tabs, so you should probably check that first before changing multirow.

**Comment by simoncarne** [comment]
@Izheil 
Many thanks for your response. Looking again at what I have installed, I realise that I am running your Multiple Tabs software in conjunction with CustomCSSforFx from Aris-t2 (see https://github.com/aris-t2/customcssforfx). Is that a project that you are aware of? 

I have tried eliminating that project altogether to see what happens with your MultiRowTab and the result is that various tab rows have different heights. 

If you **aren't** aware of the Aris-t2 project, or not happy to have yours used in conjunction with it, I will need to do some more investigating to figure out how to proceed. But, if you **are** aware of the project and are happy to work with it, I can tell you that the conflict that is affecting the tab heights seems to be coming from the file ./css/tabs/tabs_multiple_lines-classic.css. 

Any suggestions ... or do you recommend that I try to do more work on this for myself?

**Comment by Izheil** [comment]
> I have tried eliminating that project altogether to see what happens with your MultiRowTab and the result is that various tab rows have different heights.

That shouldn't be happening:
![TAB ROWS](https://user-images.githubusercontent.com/33794458/149442635-4c184417-9173-487d-b7b0-bdaadd8327a7.png)

Aris-t2 uses a different method for multirow, if you are using his multirow tabs, you shouldn't use the one here, that will most likely cause issues, and same the other way around, if using the one here, don't use the one there.

If you mean that you are just using other CSS code from that project (Knowing which css files you are using from there would help debug the issue), it shouldn't have any problem unless you were using code that is affecting the same part of the interface, which in this case, it is (the tabs area).

With that said, if you can tell me which css you are using from there, we can try to make a patch to make it work, otherwise I won't be able to help you.

**Comment by simoncarne** [comment]
Thank you for offering a patch. I will figure out which CSS I am using and get back to you. I know that I am **not** using the multi-row from Aris-t2. I am using other CSS features. 

Many thanks

**Comment by simoncarne** [comment]
I have solved the problem. Many thanks once again, @Izheil, for the advice above, which pointed me in the right direction.

In case anyone else arrives at this page in search of a solution to the same problem, it arose in my case because I was importing (from Aris-t2's CustomCSSforFx) an old version of the "tabs_multiple_lines.css" file designed for Firefox 81 (ie I was using /css/tabs/tabs_multiple_lines_**fx81**.css.) I have replaced that with the latest version of the file (ie /css/tabs/tabs_multiple_lines.css) and the problem has resolved itself.
